qt/MainWindow.h

```diff
@@ -178,6 +179,7 @@ private slots:
     QWidget* filter_bar_ = {};
     QAction* alt_speed_action_ = {};
     QString error_message_;
+    bool read_from_clipboard_ = {};
```

Comment:
_nit_ `read_from_clipboard_` as a variable name is slightly difficult to understand out of context, e.g. in a config file with no documentation. 

Just brainstorming alternatives ... possibly `auto-add-magnet-links-from-clipboard`? Maybe that's too verbose? Doesn't have to be that. Better choices are welcomed :)

Reply:
`auto-add-clipboard-magnet-links`?

Reply:
This is not limited to magnet links. Also link to the `.torrent` file works.

Behavior from user perspective is really similar to manually pressing the `Open URL`-button.
For each link separated by newline, a dialog is opened.

Maybe `auto_add_clipboard_links` or `auto_detect_clipboard_links`?